src/js/components/Bubble.js

```diff
@@ -1,18 +1,36 @@
 import React from "react"
```

Comment:
If a timer is running, the running timer is shown in the Bubble as follows:

![Bubble-with-timer](https://user-images.githubusercontent.com/871953/65500476-a501f700-debf-11e9-88a0-cf8ac05ccd4a.png)